timesketch/frontend-ng/src/components/Explore/UnfurlDialog.vue

```diff
@@ -12,66 +12,57 @@ limitations under the License.
 -->
 <template>
   <v-card>
-    <v-card-title>
-      <v-img :src="getUnfurlLogo" max-height="32" contain class="mb-2"></v-img>
-    </v-card-title>
+    <v-toolbar flat color="transparent">
+      <v-img position="left" :src="getUnfurlLogo" max-height="24" contain class="ml-2"></v-img>
+    </v-toolbar>
+
     <v-card-subtitle class="pt-1">
-      <span><b>Input:</b> {{ url }}</span>
+      <div class="mb-2"><b>Input:</b> {{ url }}</div>
+      <div v-if="nodeContext">
+        <b>Selected node info: </b>
+        <span v-html="sanitizeHtml(nodeContext)"></span>
+      </div>
     </v-card-subtitle>
+
     <v-card-text>
       <div v-show="!unfurlReady">
         <v-progress-linear color="primary" indeterminate> </v-progress-linear>
       </div>
-      <v-divider></v-divider>
-      <v-container fluid v-show="unfurlReady">
+
+      <v-card v-show="unfurlReady" outlined class="mt-n5">
+        <v-toolbar dense flat color="transparent">
```

Comment:
1. The transparent toolbar seems not to be doing anything since the graph canvas stops right below it. So this looks like a white box even with a transparent bg.
![image](https://github.com/google/timesketch/assets/99879757/78b81ddb-6505-4b67-99d7-4aeed87e3474)

2. The toolbar in the graph canvas makes the graph not controllable in this white strip which is unintuitive, since I would expect to have control on the graph in the whole box.

Reply:
I see. This is the recomeded way to lay out the controls on the right. I'll see if I can make them float instead

Reply:
Moved them outside of the box for now.